scripts/reliability/reliability_report.py

```diff
@@ -485,8 +471,8 @@ def config(env_args: os._Environ = os.environ) -> argparse.Namespace:
         "-r",
         help="DSN to connect to the Redis like service.",
         default=env_args.get(
-            "AUTOEND__RELIABILITY_DSN",
-            env_args.get("AUTOCONNECT__RELIABILITY_DSN", "redis://localhost"),
+            "AUTOCONNECT__RELIABILITY_DSN",
+            env_args.get("AUTOEND__RELIABILITY_DSN", "redis://localhost"),
```

Comment:
`--db_settings` still defaults to `AUTOEND__` before `AUTOCONNECT__`, it should be changed as well

Reply:
Yeah, I was debating that. This one is actively used, so I prioritized, but you're right, I should do all of them for consistency .